dcrtimed/backend/filesystem/filesystem.go

```diff
@@ -588,6 +588,70 @@ func (fs *FileSystem) getDigest(ts int64, current *leveldb.DB, digest [sha256.Si
 		}
 	}
 
+	// Lookup in previous not flushed dirs
+	// Get Dirs.
+	files, err := ioutil.ReadDir(fs.root)
```

Comment:
This appears to be checking against every digest ever submitted.  What are the performance impliations of this?  Why can't this be more targeted?

Reply:
Not exactly, we iterate over files which are sorted by creation time and once we find a flushed dir we stop. so this shouldn't be that expensive.

Reply:
Oh I see.  How many unflushed directories will there be at any one time?

Reply:
Ideally one, but could be a bit more, as if anchoring fails for a dir(wallet error on any other unexpected error) it will be included in next anchoring cycle.

Reply:
Makes sense.